packages/node/src/yargs.ts

```diff
@@ -9,6 +9,9 @@ import yargs from 'yargs/yargs';
 export const yargsOptions = yargs(hideBin(process.argv))
   .command({
     command: 'force-clean',
+    describe:
+      'Force cleans the database, dropping project schemas and tables (Once the command is executed, the application would exit upon completion)',
```

Comment:
@deverka can you please review this. This gets displayed when the user does `subql-node --help`